### DISCUSSION TITLE
Video tutorials

### DISCUSSION THREAD

**Discussion by ralphilius** [kalepail/stellar-quest-bounties#83] [Bounty Ideas]
Some people are good at reading, some at watching videos. There's a bunch of video sessions on Youtube in a walkthrough manner for certain tutorials. We should have this type of videos produced as a bounty.

Some ideas for the videos:

- Explain Like I'm 5, everything about Stellar
- Simple projects to consume commonly-used Stellar operations in different languages
- Complicated real-life projects using Stellar's SEP

**Comment by tourBermuda** [comment]
@samconnerone might be interested in working with someone on this.

**Comment by kalepail** [comment]
Love this. Great bounty idea. Really like that it's non-engineery. Make it happen!

**Reply by ralphilius** [reply]
Some thoughts before I make it. 

There might be different levels based on the difficulty of the tutorials. Should we make it accordingly to bounty levels? If yes, we need a clear guideline for submitters to know if their tutorial is worth at which level.

Maybe we can firstly make it simple tutorials that is level 1 bounty and see how it goes.

**Reply by kalepail** [reply]
> Maybe we can firstly make it simple tutorials that is level 1 bounty and see how it goes.

Seems reasonable.
